src/htaccess

```diff
@@ -738,6 +738,13 @@ AddEncoding br .br
     RewriteCond %{REQUEST_FILENAME} !-d
     RewriteCond %{REQUEST_URI} !\.(json|wasm|js|css|png|jpg|map)(\.(gz|br))?$
```

Comment:
Add `well-known` or the file extensions (.json, .txt) you're interested in to this regex and you'll get what you want.